src/review/Review.unit.tsx

```diff
@@ -22,7 +23,7 @@ function assertReview(
   expect(title.text()).toBe('Review title')
   const paragraph = review.find('p')
   expect(paragraph.text()).toBe('Review content')
-  const date = review.find('.kirk-caption')
```

Comment:
Can you grep `kirk-caption` making sure it's not used in other component?

Reply:
I must also update snapshots on BBC App